experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -40,6 +54,22 @@ function root_system(fam::Symbol, rk::Int)
   return R
 end
 
+@doc raw"""
+    root_system(type::Vector{Tuple{Symbol,Int}}) -> RootSystem
+
+Construct the root system of the given type. See `cartan_matrix(fam::Symbol, rk::Int)` for allowed combinations of tuples.
+
+# Examples
+```jldoctest
+julia> root_system([(:A, 2), (:F, 4)])
+Root system of rank 6
+  of type A2 x F4
```

Comment:
I'm probably missing something, but would it make sense to just print the type (if it is known)?
```suggestion
julia> root_system([(:A, 2), (:F, 4)])
A2 x F4
```

Reply:
I personally don't like it, as it introduces some kind of ambiguity: `A2 x F4` could mean the root system of that type, the Weyl group of that type, a Lie algebra with a root system of that type (where we don't print the coefficient field), or even the root lattice of that type. Thus, I am explicit here and add the information that we have a root system here.
The rank is included since we don't always have a type, and for some types (like `A5 x A5 x A3 x A2 x A5 x A3 x A3 x A5 x A5 x A2`) it is hard to read it off from the other data.
And what I kept in mind from the three-printing-mode-discussion: julia does some weird things if the "detailed" printing is multi-line for some objects of the type and single-line for others. Thus I tried to come up with something here that always has 2 lines.